cloudformation/master-server-substack.cfn.yaml

```diff
@@ -492,6 +535,10 @@ Resources:
                 - !Ref 'ProxyServer'
                 - 'false'
     Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - E3002
```

Comment:
Why do we need to add these values?

if it's to fix travis we can add this in the cfn_template_rendering code:
```python
    cfn_lint_args = ["--info", str(rendered_file), "-i", "E3002" ]
    with patch.object(sys, "argv", cfn_lint_args):
        assert cfnlint() == 0
```

Reply:
We decided to keep this ignore here to avoid to fail for all the templates.